lib/app/models/sequence_result.rb

```diff
@@ -84,9 +100,9 @@ def update_result_counts
             end
           when ResultStatuses::SKIP
             if result.required
-              self.skip_count += 1
               self.result = result.result if pass?
             end
+            self.skip_count += 1
```

Comment:
I assume the skip_count should be counting all skips, not just the required ones.  If not we should rename it.  I did a "find in files" for all references for skip_count, and this change does not really affect anything because we only really use it for display.

Reply:
I remember discussing this previously and I think we landed on only counting required skips even though the total wouldn't add up. 
 See FI-157 and PR-157.  @arscan might want to weigh in to make sure there won't be unintended consequences.

Reply:
ok for now I can change it back, but should we consider renaming it in case we later want to track optional_skips or total_skips?